### DISCUSSION TITLE
Preview successfully deployed but github blocks merging

### DISCUSSION THREAD

**Discussion by anya-ji** [vercel/community#4842] [Help] [answered]
### Summary

When merging a branch, Preview seems to be successfully deployed, but GitHub blocks merging saying Missing successful Preview deployment. In the settings, I think I selected the correct Preview (the other 2 environments were created by mistake and now deleted). So what am I missing?

### Steps to Reproduce

PR blocking merge:
<img width="923" alt="Screen Shot 2023-11-13 at 10 44 19" src="https://github.com/vercel/community/assets/61763411/53695c3d-2e1d-4246-a5df-3b02a66972d4">

Master branch setting:
<img width="740" alt="Screen Shot 2023-11-13 at 10 46 42" src="https://github.com/vercel/community/assets/61763411/edfa4c3f-8407-416d-99b7-b07aa4b84cee">

**Comment by amyegan** [comment, marked as answer]
Hi @anya-ji. This situation sounds similar to https://github.com/vercel/community/discussions/3215#discussioncomment-6969235. Please double check your GitHub settings to make sure the deployment environment being checked matches the deployment environment being used.
